plugins/indexing/aws/sqs_client.go

```diff
@@ -0,0 +1,69 @@
+package pluginaws
```

Comment:
Hm I thought package name always had to match the directory name (`aws` in this case) in golang..

Reply:
I checked that it's just a convention. Up to you I guess ultimately..

Reply:
So true what they say about variable naming!

Reply:
Upon checking Geth I believe we should follow the convention and name the package `aws` even if it sounds generic.